BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudLobby.cs

```diff
@@ -385,27 +385,46 @@ public void GetRegionsForLobbies(string[] in_roomTypes, SuccessCallback success
-		 *
-		 * Service Name - Lobby
-		 * Service Operation - GetVisibleLobbyInstances
-		 *
-		 * @param lobbyType The type of lobby to look for.
-		 * @param minRating Minimum lobby rating.
-		 * @param maxRating Maximum lobby rating.
-		 */
-        public void GetVisibleLobbyInstances(string in_roomType, int in_minRating, int in_maxRating, SuccessCallback success = null, FailureCallback failure = null, object cbObject = null)
+         * Gets a map keyed by rating of the visible lobby instances matching the given type and rating range.
+         * any ping data provided in the criteriaJson will be ignored.
+         *
+         * Service Name - Lobby
+         * Service Operation - GetLobbyInstances
+         *
+         * @param lobbyType The type of lobby to look for.
+         * @param criteriaJson A JSON object used to describe filter criteria.
+         */
+        public void GetLobbyInstances(string in_lobbyType, Dictionary<string, object> criteriaJson, SuccessCallback success = null, FailureCallback failure = null, object cbObject = null)
         {
             Dictionary<string, object> data = new Dictionary<string, object>();
-            data[OperationParam.LobbyRoomType.Value] = in_roomType;
-            data[OperationParam.LobbyMinRating.Value] = in_minRating;
-            data[OperationParam.LobbyMaxRating.Value] = in_maxRating;
+            data[OperationParam.LobbyRoomType.Value] = in_lobbyType;
+            data[OperationParam.LobbyCritera.Value] = criteriaJson;
 
             ServerCallback callback = BrainCloudClient.CreateServerCallback(success, failure, cbObject);
-            ServerCall sc = new ServerCall(ServiceName.Lobby, ServiceOperation.GetVisibleLobbyInstances, data, callback);
+            ServerCall sc = new ServerCall(ServiceName.Lobby, ServiceOperation.GetLobbyInstances, data, callback);
             m_clientRef.SendRequest(sc);
         }
         
+        /**
+         * Gets a map keyed by rating of the visible lobby instances matching the given type and rating range.
+         * Only lobby instances in the regions that satisfy the ping portion of the criteriaJson (based on the values provided in pingData) will be returned.
+         *
+         * Service Name - Lobby
+         * Service Operation - GetLobbyInstancesWithPingData
+         *
+         * @param lobbyType The type of lobby to look for.
+         * @param criteriaJson A JSON object used to describe filter criteria.
+         */
+        public void GetLobbyInstancesWithPingData(string in_lobbyType, Dictionary<string,object> criteriaJson,
+            SuccessCallback success = null, FailureCallback failure = null, object cbObject = null)
+        {
+            Dictionary<string, object> data = new Dictionary<string, object>();
+            data[OperationParam.LobbyRoomType.Value] = in_lobbyType;
+            data[OperationParam.LobbyCritera.Value] = criteriaJson;
```

Comment:
You're missing the ping data, that is stored internally in the LobbyService class.

Reply:
wait let me check how that used to be done...